browser/components/MarkdownPreview.js

```diff
@@ -832,6 +832,82 @@ export default class MarkdownPreview extends React.Component {
         )
       }
     )
+
+    document.querySelectorAll('iframe').forEach(item => {
+      const rect = item.getBoundingClientRect()
+      const imgList = item.contentWindow.document.body.querySelectorAll('img')
+      for (const img of imgList) {
+        img.onclick = () => {
+          const widthMagnification = document.body.clientWidth / img.width
+          const heightMagnification = document.body.clientHeight / img.height
+          const baseOnWidth = widthMagnification < heightMagnification
+          const magnification = baseOnWidth ? widthMagnification : heightMagnification
+
+          const zoomImgWidth = img.width * magnification
+          const zoomImgHeight = img.height * magnification
+          const zoomImgTop = document.body.clientHeight / 2 - zoomImgHeight / 2
```

Comment:
Little suggestion, you can transform this code into:
```js
const zoomImgTop = (document.body.clientHeight - zoomImgHeight) / 2
```
This way, we only need to do 2 calculation instead of 3

Reply:
I didn't notice it.
I am going to fix it tomorrow. 
Thank you for teaching it!

Reply:
No problem :+1: